src/site/xdoc/usage.xml

```diff
@@ -466,145 +465,141 @@ catch (ParseException exp) {
               <code>deprecated</code> method.
 
               Usage of the deprecated option is announced when the presence of the option is checked
-              or the value of the option is retrieved.  By default the announcement printed to Standard out.
+              or the value of the option is retrieved.  By default, the announcement printed to Standard out.
 
-              The HelpFormatter output will be default show the description prefixed by "[Deprecated]"
+              The HelpFormatter output will by default show the description prefixed by "[Deprecated]"
           </p>
           <p>
               The examples below will implement <code>doSomething</code> in the following code block.
               <source>
-                  public static void main(String[] args) {
-                      Option n = Option.builder("n")
-                          .deprecated(DeprecatedAttributes.builder()
-                              .setDescription("Use '-count' instead")
-                              .setForRemoval(true)
-                              .setSince("now").get())
-                          .hasArg()
-                          .desc("the number of things")
-                          .type(Integer.class)
-                          .build();
-                      Option count = Option.builder("count")
-                          .hasArg()
-                          .desc("the number of things")
-                          .type(Integer.class)
-                          .build();
-                      Options options = new Options().addOption(n).addOption(count);
+  public static void main(String[] args) {
+      Option n = Option.builder("n")
+          .deprecated(DeprecatedAttributes.builder()
+              .setDescription("Use '-count' instead")
+              .setForRemoval(true)
+              .setSince("now").get())
+          .hasArg()
+          .desc("the number of things")
+          .type(Integer.class)
+          .build();
+      Option count = Option.builder("count")
+          .hasArg()
+          .desc("the number of things")
+          .type(Integer.class)
```

Comment:
Separate discussion: Should we add common helpers like:
`.interger()` as a shorthand for `.type(Integer.class)`?

Reply:
Check.